pages.ko/linux/add-apt-repository.md

```diff
@@ -0,0 +1,20 @@
+# add-apt-repository
+
+> Manages apt repository definitions.
```

Comment:
Thanks for your contribution, this line seems to be untranslated. Please translate it and update the page.